app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/tiles/GeneratorTileService.kt

```diff
@@ -0,0 +1,53 @@
+package com.x8bit.bitwarden.ui.platform.feature.tiles
```

Comment:
Our other services are declared in a `data` sub-package. @david-livefront That's where these should be declared, right?

Reply:
That is correct, this should not be in the `ui` package.